src/algorithm/destination.rs

```diff
@@ -0,0 +1,62 @@
+use num_traits::{Float, FromPrimitive};
+use types::Point;
+
+/// Returns a new Point using the distance to the existing Point and a bearing for the direction
+
+pub trait Destination<T: Float> {
+    /// Returns a new Point using distance to the existing Point and a bearing for the direction
+    ///
+    /// ```
+    /// # extern crate geo;
+    /// # #[macro_use] extern crate approx;
+    /// #
+    /// use geo::Point;
+    /// use geo::algorithm::destination::Destination;
+    ///
+    /// # fn main() {
```

Comment:
once you remove the three lines above, you can remove this `fn main() {` and closing `}`